index.html

```diff
@@ -122,88 +127,8 @@ <h5 class="card-title text-danger fw-bold">CC Ignite</h5>
 
     <section class="container-fluid bg-light m-5 p-5">
         <h3 class="text-center">Featured Speakers</h3>
-        <div class="row px-5 d-flex justify-content-center">
-            <div class="col-md-6 my-5">
-                <div class="row">
-                <img src="/images/farhan.svg" class="col-md-6 rounded-circle" alt="M.Farhan a motivational speaker" height="200px">
```

Comment:
As recommended by the previous reviewer it's important to follow the original design idea, so please adjust the margin for the speaker's section. And also the speaker photos are **squared** not **rounded**, so please make sure to update that as well.

Your design
![image](https://user-images.githubusercontent.com/34779928/211147417-a287c7ee-3d6f-4380-9986-66bdb9a81e95.png)

Original design

![image](https://user-images.githubusercontent.com/34779928/211147410-1fa74abd-cd8c-4182-9af6-7f15e7ba9fa9.png)

Reply:
@aimalamiri 
1. Thanks for your  kind review but I have already applied margin for the "featured  peakers".
2. No need to strictly follow the design as said in assignment page. 
So please review again and let me submit it.